lib/screens/shuttle_schedule.dart

```diff
@@ -13,18 +26,486 @@ class ShuttleSchedule extends StatelessWidget {
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('20:00'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('20:10'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('20:30'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('20:30'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('20:45'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('21:00'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('21:05'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('21:25'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('21:30'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('21:45'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('22:00'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('22:00'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('22:30'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('22:30'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('23:00'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('23:00'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
```

Comment:
In the next iteration, we should refactor this to use [shuttleSchedule.json](/MichelRahme/ConcordiaNavigation/blob/feature/78-shuttleSchedule/assets/shuttleSchedule.json)